src/ui-kit/form-templates/international-phone/sam-extension/extension.component.ts

```diff
@@ -40,7 +44,15 @@ export class SamExtension extends SamFormControl {
    */
    placeholder: string = 'ex: 1234';
   
-
+   /**
+    * is extension required
+    */
+   public allowChars = [
```

Comment:
@yerramshilpa Can you make this private?

Reply:
Also, you're using this in three places. Can you put it in the root file for the phone extension in a file called `number-input-keys.ts` and import into all three places? Inside there should be a const you export:

```
export const numberInputKeys = [<keys>];
```